### PR TITLE
[RSS-11] New Account Page

### PR DESCRIPTION
**Pull request by esauerbo**
Created a page for making new accounts with fields for name, email, and phone number. If any of the fields are blank upon submission the user is alerted with a window alert as well as text on the page that tells them which field needs to be filled out. The data is stored as a state variable.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (jackfales) [inline .vscode/launch.json]: Remove this file

File: package.json [hunk 2]
```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@progress/kendo-licensing": "^1.1.3",
+    "@progress/kendo-react-data-tools": "^4.3.0",
+    "@progress/kendo-react-form": "^4.3.0",
+    "@progress/kendo-react-intl": "^4.3.0",
```
Comment (jackfales) [inline package.json]: What are these packages used for?

File: src/components/NewAccountPage/NewAccountPage.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+// import { Button } from 'react-bootstrap';
+// import styled from 'styled-components';
+
+/* const SubmitButton = styled(Button)`
+  background-color: #6593D6;
+  color: white;
+  padding: 10px;
+  border-radius: 10px;
+  border: none;
+`; */
```
Comment (jackfales) [inline src/components/NewAccountPage/NewAccountPage.jsx]: Remove these extra comments if they're unused
```suggestion
```

File: package.json (hunk 2 above)
Comment (esauerbo) [reply package.json]: I had been trying out different packages for error handling but they're unused should I delete them?

File: src/components/NewAccountPage/NewAccountPage.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+class NewAccountPage extends React.Component {
```
Comment (jackfales) [inline src/components/NewAccountPage/NewAccountPage.jsx]: Can you convert this to a functional component? We will talk about it during the standup.

**Review by jackfales — CHANGES_REQUESTED** [review]
Check comments. Good job following proper formatting for the PR.

File: src/components/NewAccountPage/NewAccountPage.jsx (hunk 4 above)
Comment (comalvirdi) [reply src/components/NewAccountPage/NewAccountPage.jsx]: @esauerbo lmk if you need any help with this

**Comment by comalvirdi** [comment]
Looks good, only change that needs to be made is turning the component into a functional component! Good job!

File: package.json (hunk 2 above)
Comment (jackfales) [reply package.json]: Yea I will thinking you use state for error handling. Let me know if you need help and we can set up a call @esauerbo

File: .eslintcache [hunk 5]
```diff
@@ -0,0 +1 @@
...tedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/emmasauerborn/rise-stand-strong/src/index.js",[],["23","24","25","26","27","28"],"/Users/emmasauerborn/rise-stand-strong/src/App.js",[],"/Users/emmasauerborn/rise-stand-strong/src/components/Main/Main.jsx",[],"/Users/emmasauerborn/rise-stand-strong/src/components/LoginPage/LoginPage.jsx",[],"/Users/emmasauerborn/rise-stand-strong/src/components/NewAccountPage/NewAccountPage.jsx",[],{"ruleId":"29","replacedBy":"30"},{"ruleId":"31","replacedBy":"32"},{"ruleId":"33","replacedBy":"34"},{"ruleId":"35","replacedBy":"36"},{"ruleId":"37","replacedBy":"38"},{"ruleId":"39","replacedBy":"40"},"lines-around-directive",["41"],"no-spaced-func",["42"],"global-require",[],"no-buffer-constructor",[],"no-new-require",[],"no-path-concat",[],"padding-line-between-statements","func-call-spacing"]
```
Comment (jackfales) [inline .eslintcache]: Remove this file

**Review by jackfales — APPROVED** [review]
Looks good!

File: .eslintcache (hunk 5 above)
Comment (jackfales) [reply .eslintcache]: Nvm I fixed the conflict

**Pull request merged**